style.css

```diff
@@ -0,0 +1,51 @@
+@media screen and (max-width: 1024px) {
+    * { 
+        box-sizing: border-box;
+        margin: 0px;
+        padding: 0px;
+    }
+
+    body {
+        color: black;
+        background-color: rgb(252, 250, 248);
+    }
+
+    .nav-bar {
+        margin-top: 50px;
+        background-color: aqua;
+    }
+
+    h3{
+        font-size: 1.7rem;
+    }
+    
+    h1{
+        font-size: 2rem;
+        line-height: 0.9;
+    }
+    .content1 h3,h1 {
+        color: rgb(243, 97, 44);
+        margin-left: 25px;
+        font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande';
+    }
+    
+    .hamburger {
+        display: block;
+        padding-left: 30px;
+        color: aqua;
+        background-color: brown;
+    }
+    .content1 {
+        text-align: justify;
+    }
+    .content1 p {
+        background-color: rgb(252, 248, 246);
+        font-size: 1rem;
+        margin-left: 40px;
+        margin-top: 30px;
```

Comment:
As part of the requirement for this milestone is to make sure your website looks like the given design, I think your text sizes are smaller than the design,I'll suggest you go through the design and adjust your sizes appropriately. 😄 👍

 _Your website should look exactly like the design (e.g. font, colors, images, text, margins,backgroud)_

<ul>
<li>REQUIRED DESIGN:</li>
![DataLive - Google Chrome 11_12_2023 00_22_41](https://github.com/Mabella29/testn/assets/102887817/a9caae81-71dd-441e-89f6-db187bb637ea)


<li>YOUR CURRENT DESIGN:</li>
![Blog - Google Chrome 19_12_2023 10_03_30](https://github.com/Mabella29/testn/assets/102887817/430ce06f-57fd-4062-86b0-c903867fa6c7)
</ul>